### PR TITLE
Move CI from Travis to GitHub

### PR DESCRIPTION
**Pull request by jrmajor**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+  - pull_request
+  - push
+
+jobs:
+  tests:
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - phpunit-version: 4
```
Comment (keradus) [inline .github/workflows/ci.yml:13]: we claim `^4.8.36` , not ^4. can we narrow it down, here?

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+            php-version: '5.5'
+          - phpunit-version: 4
+            php-version: '5.6'
+          - phpunit-version: 5
+            php-version: '5.6'
+          - phpunit-version: 5
```
Comment (keradus) [inline .github/workflows/ci.yml:19]: initial solution was iterating over each PHP Version and each PHPUnit version, if they are compatible - run the tests.

now, we hardcoded the valid combinations, hoping we will never forget to add a valid variation.I think it's error-prone , but we also don't have so many changes in this repo, so not blocking

File: .github/workflows/ci.yml (hunk 1 above)
Comment (jrmajor) [reply .github/workflows/ci.yml:13]: `composer update phpunit/phpunit:^4` will respect both constraint from `composer.json` and the temporary one (in this case `^4 && ^4.8.36`).

**Comment by keradus** [comment]
I guess no other way to validate this PR than merging it. Pls raise a follow-up PR if sth goes wrong after the merge

**Comment by keradus** [comment]
kudos ;)

File: .github/workflows/ci.yml (hunk 2 above)
Comment (jrmajor) [reply .github/workflows/ci.yml:19]: If I understand the script correctly, it wasn't “if they are compatible”, but “if `composer require && composer update` does not fail”, which is also error-prone. I don't have any good idea how to do this check reliably.

**Pull request merged**